manage_config_workflow_page.php

```diff
@@ -344,8 +345,11 @@ function access_row() {
 			# 'NEW' status
 			$t_level_project = $t_project_new;
 
-			$t_can_change = ( $g_access >= config_get_access( 'report_bug_threshold' ) );
-			$t_color = set_color_override( $t_file_new, $t_global_new, $t_project_new );
+			# If report_bug_threshold is an array (instead of an integer value), the input is not editable
+			# because it must be configured in manage_config_work_threshold_page.
+			$t_can_change = ( $g_access >= config_get_access( 'report_bug_threshold' ) )
```

Comment:
config_get_access( 'report_bug_threshold' ) should not be affected with the actual value of the option, is checking "who can change this".
Actually the !is_array is for other reason, as the idea is that a threshold as an array cannot be edited as a plain dropdown select.
This commit is to not not make that field editable if its not a numeric value.